Description: leveldb is not part of this source, don't autoreconf there
 Packaging uses system leveldb and as such, src/leveldb is empty here. No need
 to do autoreconf there.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Last-Update: <2012-06-13>

---

--- ceph-0.47.2.orig/autogen.sh
+++ ceph-0.47.2/autogen.sh
@@ -19,5 +19,5 @@ autoconf
 autoheader
 automake -a --add-missing -Wall
 ( cd src/gtest && autoreconf -fvi; )
-( cd src/leveldb && autoreconf -fvi; )
+#( cd src/leveldb && autoreconf -fvi; )
 exit
